transcriptomics/transcriptomics_matrix_generator.py

```diff
@@ -45,23 +45,32 @@ def load_sample_info(self, json_file_path):
             self.treatments = [item['Treatment'] for item in sample_info]
             self.experiment_id = [item['ExperimentID'] for item in sample_info]
 
-    def download_gff(self, url, file_path):   
+    def download_gff(self, url, file_path):
```

Comment:
Would rename to `download_and_process_gff`, since it does more than just download now.